core/src/com/unciv/logic/automation/WorkerAutomation.kt

```diff
@@ -357,25 +364,24 @@ class WorkerAutomation(
 
             // Defence is more important that civilian improvements
             // While AI sucks in strategical placement of forts, allow a human does it manually
-            !civInfo.isPlayerCivilization() && evaluateFortPlacement(tile, civInfo, false) -> Constants.fort
+            !civInfo.isPlayerCivilization() && evaluateFortPlacement(tile, civInfo,false) -> Constants.fort
```

Comment:
Very minor point but basically the only thing of this PR I don't agree with :)

Reply:
That's what I get for trying to undo auto-formatting...